ext/openssl/tests/ecc.phpt

```diff
@@ -95,6 +96,76 @@ foreach ($curve_names as $curve_name) {
         echo "Found secp384r1 in curve names\n";
     }
 }
+
+// EC - generate keypair with curve_name
+echo "Testing openssl_pkey_new with ec curve_name\n";
```

Comment:
Could you please create a new test for this functionality. The build showed some failure on ASAN and it might be hard to get the right place what is failing there. I prefer more smaller tests than one big test as it is much easier to debug.

Reply:
I don't know why strange issues like memory leaks appear in ASAN and FREEBSD testing, while my local test results are normal (using OpenSSL 1.1.1m and OpenSSL 3.1.0). Could you please assist me in identifying the issue?

Reply:
There was indeed a memory leak, and I lost the `ctx` pointer at line 4572, which caused the memory leak. It has now been fixed.
However, I still don't know why running the `ext/standard/tests/general_functions/phpinfo.phpt` test on ASAN, ARM, and FREEBSD fails.